README.md

```diff
@@ -72,6 +84,74 @@ Here are some example questions you can ask the bot:
+|---------------------------------|-------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| direct mention                  | _Rasa_ is located in _Berlin_.                                                                        | An entity was mentioned by its name in the text, such as _Rasa_ or _Berlin_.                                                                                                       |
+| mention by pronoun              | _It_ is an open source company.                                                                       | Reference to a previous mentioned entity by its pronoun, such as _it_.                                                                                                             |
+| ordinal mention                 | Where is the headquarters of the _third_ company?                                                     | The user was confronted to a list of entities. The user refers to an entity of that list by its position.                                                                          |
+| mention by attribute(s)         | Where is the headquarters of the _open source_ company that _builds chatbots_?                        | Find an entity in the knowledge base that fits the mentioned criteria.                                                                                                             |                                                                                 
+| ambiguous mention               | I want to transfer money to _John_?                                                                   | The bot can find multiple entities with the referenced name (e.g. _John_). The entity needs to be specified.                                                                       |                                                                                 
+| mention by synonym              | What's my _balance_? vs. How much _cash_ do I have?                                                   | The user refers to the same entity with different names.                                                                                                                           |                                                                                 
+| mention by hypernyms & hyponyms | Here are some recent transactions: Deutsche Bahn 99.95€, Spotify 9.99€. Which do you want to dispute? | The user can answer many things, such as "the bigger one", "the subscription", or "the train ticket". The bot needs to understand what entity the user refers to by those answers. |                                                                                 
```

Comment:
"the bigger one" is by attribute comparison, which I'd say is a separate category. 
maybe we just do a travel example and pick something  that's international-friendly (I don't think everyone will know  what Deutsche  Bahn is )